### PR TITLE
Add pnpm as an option other than npm

### PR DESCRIPTION
**Pull request by quirinpa**
Tests of execution are here:
- [core 173](https://github.com/MOV-AI/frontend-npm-lib-core/pull/173) :large_green_circle:
- [lib-react 275](https://github.com/MOV-AI/frontend-npm-lib-react/pull/275) :large_green_circle:
- [scene 245](https://github.com/MOV-AI/frontend-npm-lib-scene/pull/245) :large_green_circle:
- [ide 272](https://github.com/MOV-AI/frontend-npm-lib-ide/pull/272) :large_green_circle:
- [adminboard 148](https://github.com/MOV-AI/frontend-npm-adminboard/pull/148) :large_green_circle:
- [fleetmanager 304](https://github.com/MOV-AI/frontend-npm-fleetmanager/pull/304) :large_green_circle:
- [ide 524](https://github.com/MOV-AI/frontend-npm-ide/pull/524) :large_orange_circle:
- [taskmanager 129](https://github.com/MOV-AI/frontend-npm-taskmanager/pull/129) :large_green_circle:
- [launcher 113](https://github.com/MOV-AI/frontend-npm-launcher/pull/113) :large_green_circle:

Adds pnpm as alternate package manager

### CODE REVIEW

**Review by AlexFernandes-MOVAI — COMMENTED** [review]
please provide a description and working pipeline execution

File: .github/workflows/frontend-workflow.yml [hunk 1]
```diff
@@ -70,20 +74,27 @@ jobs:
...
+    - name: Setup Github as proxy Registry
+      uses: actions/setup-node@v2
+      with:
+        node-version: '${{ inputs.npm_version }}'
+        registry-url: 'https://npm.pkg.github.com/mov-ai'
+
+    - name: Setup .npmrc and install pnpm
+      if: ${{ inputs.pm == 'pnpm' }}
+      run: |
+        npm install -g pnpm
```
Comment (duartecoelhomovai) [inline .github/workflows/frontend-workflow.yml:86]: I would suggest creating a new version of the agent where this runs and have this installs there.

Comment (duartecoelhomovai) [reply .github/workflows/frontend-workflow.yml:86]: I can help on that

Comment (duartecoelhomovai) [reply .github/workflows/frontend-workflow.yml:86]: Lets not merge this into v2. 
I will create a new branch v2.1, and that will be the target branch

**Review by duartecoelhomovai — CHANGES_REQUESTED** [review]

File: .github/workflows/frontend-workflow.yml (hunk 1 above)
Comment (duartecoelhomovai) [reply .github/workflows/frontend-workflow.yml:86]: qa/npm-buildserver:v1.0.0 has it

Comment (quirinpa) [reply .github/workflows/frontend-workflow.yml:86]: Is this correct?
      image: registry.aws.cloud.mov.ai/qa/npm-buildserver:v1.0.0

**Review by duartecoelhomovai — APPROVED** [review]

**Pull request merged**